dydx/dydxAnalytics/dydxAnalytics/AnalyticsEvent.swift

```diff
@@ -67,76 +67,85 @@ public extension AnalyticsEventV2 {
     }
 }
 
-public enum AnalyticsEventV2: TrackableEvent, CustomDebugStringConvertible {
-    // TODO: add and replace all events
-    case appStart
-    case navigatePage(screen: ScreenIdentifiable)
-    case deepLinkHandled(url: String, succeeded: Bool)
-    case notificationPermissionsChanged(isAuthorized: Bool)
-    case onboardingStepChanged(step: OnboardingStep, state: OnboardingState)
-
-    public var name: String {
-        switch self {
-        case .navigatePage:
-            return "NavigatePage"
-        case .deepLinkHandled:
-            return "DeeplinkHandled"
-        case .appStart:
-            return "AppStart"
-        case .onboardingStepChanged:
-            return "OnboardingStepChanged"
-        case .notificationPermissionsChanged:
-            return "NotificationPermissionsChanged"
+public enum AnalyticsEventV2 {
```

Comment:
@ruixhuang note I am nesting them in `AnalyticsEventV2` so that we can use Xcode's autofill after typing `AnalyticsEventV2.`, also to keep them contained. Lmk if u think this is not worth it and I can put them at the global level.